src/app/log_window.rs

```diff
@@ -109,25 +79,25 @@ impl LogWindow {
                             Err(p) => self.save_promise = Some(p),
                         }
                     } else if ui.button("Save to file").clicked() {
-                        self.buffer.make_contiguous();
-                        let buffer = self.buffer.clone();
-
-                        self.save_promise = Some(luminol_core::spawn_future(async move {
-                            use futures_lite::AsyncWriteExt;
-
-                            let mut tmp = luminol_filesystem::host::File::new()?;
-                            let mut cursor = async_std::io::Cursor::new(buffer.as_slices().0);
-                            async_std::io::copy(&mut cursor, &mut tmp).await?;
-                            tmp.flush().await?;
-                            tmp.save("luminol.log", "Log files").await?;
-                            Ok(())
-                        }));
+                        // self.buffer.make_contiguous();
+                        // let buffer = self.buffer.clone();
+
+                        // self.save_promise = Some(luminol_core::spawn_future(async move {
+                        //     use futures_lite::AsyncWriteExt;
+
+                        //     let mut tmp = luminol_filesystem::host::File::new()?;
+                        //     let mut cursor = async_std::io::Cursor::new(buffer.as_slices().0);
+                        //     async_std::io::copy(&mut cursor, &mut tmp).await?;
+                        //     tmp.flush().await?;
+                        //     tmp.save("luminol.log", "Log files").await?;
+                        //     Ok(())
+                        // }));
                     }
```

Comment:
Don't forget to implement this, or at least remove the button or provide some other way of copying log text